test/powershell/Modules/Microsoft.PowerShell.Core/Invoke-Command.Tests.ps1

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+Describe "Invoke-Command" -Tags "CI" {
+    Context "StrictModeVersion tests" {
+        BeforeAll {
+            $skipTest = ![ExperimentalFeature]::IsEnabled("PSStrictModeVersionAssignment")
+            If (Test-Path Variable:InvokeCommand__Test) {
+                Remove-Item Variable:InvokeCommand__Test
+            }
+        }
+
+        It "Setting -StrictModeVersion parameter with uninitialized variable throws error"  -skip:$skipTest {
+            { Invoke-Command -StrictModeVersion 3.0 {$InvokeCommand__Test} } | Should -Throw -ErrorId 'VariableIsUndefined'
+        }
+
+        It "Setting -StrictModeVersion parameter with initialized variable does not throw error" -skip:$skipTest {
+            $InvokeCommand__Test = 'Something'
+            Invoke-Command -StrictModeVersion 3.0 {$InvokeCommand__Test} | Should -Be 'Something'
```

Comment:
Please add a test verifying that `Invoke-Command -StrictModeVersion latest` works. `Set-StrictMode -Version latest` works.

Reply:
We should also test that strictmode can be turned off with null value, or we will need to also add a `-StrictModeOff` parameter.

Reply:
Is strictmodeversion 0.0 essentially off?

Reply:
Yep, that's how `Set-StrictMode -Off` works.